integration/flink/examples/stateful/build.gradle

```diff
@@ -58,17 +58,18 @@ ext {
     icebergVersion = "1.3.0"
 
     versionsMap = [
-            "1.15": ["cassandra": "1.15.4", "kafka": flinkVersion, "alternativeShort": "1.15"],
-            "1.16": ["cassandra": "3.0.0-1.16", "kafka": flinkVersion, "alternativeShort": "1.16"],
-            "1.17": ["cassandra": "3.1.0-1.17", "kafka": flinkVersion, "alternativeShort": "1.17"],
-            "1.18": ["cassandra": "3.1.0-1.17", "kafka": "3.0.2-1.18", "alternativeShort": "1.17"]
+            "1.15": ["cassandra": "1.15.4", "kafka": flinkVersion, "jdbc": "1.15.4", "alternativeShort": "1.15"],
+            "1.16": ["cassandra": "3.0.0-1.16", "kafka": flinkVersion, "jdbc": "1.16.3", "alternativeShort": "1.16"],
+            "1.17": ["cassandra": "3.1.0-1.17", "kafka": flinkVersion, "jdbc": "3.1.1-1.17", "alternativeShort": "1.17"],
+            "1.18": ["cassandra": "3.1.0-1.17", "kafka": "3.0.2-1.18", "jdbc": "3.1.1-1.17", "alternativeShort": "1.17"]
     ]
 
     versions = versionsMap[flinkVersionShort]
 }
 
 dependencies {
-    implementation project(':')
+    implementation(project(path: ":app"))
```

Comment:
I'm concerned that we're going to run into a cyclic dependency here. In the `app/build.gradle` you're asking for the output of ':extamples:stateful:shadowJar', but over here, you're asking for the JAR of `app`.

Reply:
It is actually not cyclic dependency. Example depends on app in terms of lib. App's shadow jar depends on example's shadow jar.